test/base/src/macosTest/kotlin/io/realm/test/CoroutineTests.kt

```diff
@@ -49,8 +49,13 @@ class CoroutineTests {
     // Fails on non native-mt as dispatching from background thread to main does change actual
     // thread, thus failing to assert main thread id
     @Test
+    // Ignored for new memory model as (from https://github.com/JetBrains/kotlin/blob/master/kotlin-native/NEW_MM.md):
```

Comment:
Is there an intent to do something about this? With the new test changes in 1.6.0, I think it becomes possible to override the Main dispatcher on all platforms.

Reply:
I added a comment to #634 on addressing this